pkg/api/persistentvolume/util.go

```diff
@@ -32,6 +33,15 @@ func DropDisabledFields(pvSpec *api.PersistentVolumeSpec, oldPVSpec *api.Persist
 	}
 }
 
+// AddDisabledFieldsTo to the provided set of resetFields
+// This should be called with the resetFields for all resources containing a Pod
+func AddDisabledFieldsTo(resetFields *fieldpath.Set) {
+	if !utilfeature.DefaultFeatureGate.Enabled(features.ExpandCSIVolumes) {
+		// TODO: make sure this path is correct, the api type has some inlines
```

Comment:
fix TODO?

Reply:
@apelisse, do you remember what the conclusion we came to on these TODOs? There's quite a few peppered in here still. Do you know which have already been fixed vs want to leave in as todos vs need fixed for this release?

Reply:
Some of these can't be fixed, some of these should be double checked, some of these might be fixable.

Reply:
Also, it's important to acknowledge that we know this isn't the perfect solution and we won't have all of these rights, the more the better, but we certainly don't want to block on that.

Reply:
The comment makes it sound like we aren't sure if the path actually references a field at all -- I don't think we should have any errors of that sort.

Whether we've found all the fields that need this treatment is a different question.

Reply:
I agree with that.